uui/components/filters/FilterNumericBody.tsx

```diff
@@ -66,7 +66,7 @@ export function FilterNumericBody(props: IFilterNumericBodyProps) {
         const value = props.value as INumericRangeValue;
         return (
             <div>
-                <FlexRow padding="12" vPadding="24" alignItems="center" spacing="12" borderBottom={ true } rawProps={ { style: { paddingTop: '3px' } } }>
+                <FlexRow padding="12" alignItems="center" spacing="12" borderBottom={ true } rawProps={ { style: { marginTop: '12px', paddingBottom: '12px' } } }>
```

Comment:
let's get rid of inline styles and move it to the css classes

Reply:
done